src/olympia/amo/tests/test_amo_utils.py

```diff
@@ -178,7 +178,7 @@ def test_to_language(test_input, expected):
         ('en_US', 'en-US'),
         ('en', 'en-US'),
         ('FR', 'fr'),
-        ('es-ES', None),  # We don't go from specific to generic.
+        ('fr-FR', None),  # We don't go from specific to generic.
```

Comment:
Such a large PR, is there a reason to go from spanish to french here?

Reply:
Yes, because this test checks that `xx-XX` when we only have a `xx` locale returns `xx`. We are introducing `es-ES` (and removing `es`) so I had to change it to something else.